charter/index.html

```diff
@@ -328,16 +344,16 @@ <h3>
             </div>
 
             <div id="timeline">
-                <h3>Timeline (TBD)</h3>
+                <h3>Timeline <i class=issue>(TBD)</i></h3>
```

Comment:
Do you have any suggestions as to what should go here? It seemed that in other charters this section typically had explicit dates that began at the working group's start date, and so this is TBD since we don't know when that is yet.

Reply:
common practice is to use relative deadlines, like "START + 12M", "START + 18M"

Reply:
```suggestion
                <h3>Timeline <i class=todo>(TBD)</i></h3>
```

Reply:
I'll put these in a followup PR shortly.